python/src/aac/spec/spec.yaml

```diff
@@ -340,6 +340,9 @@ schema:
         - name
```

Comment:
Based on the PRs leverage, updates to spec to apply the validator was how AC like:
 `- The new validator is applied to the `name` field in the schema definition
     - All other definitions should inherit the validation and have it applied to them during validation` 
 are being met.

Reply:
This looks good! I was a pretty vague in the AC, but we don't need to have the argument for the 'name' field. The suggested edit for the validator implementation assumes that the name field is present and it's not leveraging the arguments to indicate which definition field contains the value that's being used to identify references by other definitions -- we assume that's the name field as we solely use definition names as references between them.

Reply:
Glad I was justified in "warring" with myself on this since the implementation was all working with the name field, but didn't understand all the pieces well enough to know for certain. So the other validator that focused on specific fields was the required fields so mimicked it. 

Thanks for the clarifying explanation!